Azure/azure-cli
# [AKS] `aks create\aks nodepool add`: Add parameter ` --os-sku` to support choosing the underlying container host OS

**Description**

AKS has added the ability to choose the underlying container host OS for AKS nodepools, the `--os-sku` argument is a string that tells the service which OS to select. We're GAing CBLMariner as an AKS container host for internal use at Microsoft shortly, so we need to move the `--os-sku` argument from the aks-preview extension into the az-cli acs module.

This is a direct port of changes that have been live for several months without issue in the [aks-preview extension](https://github.com/Azure/azure-cli-extensions/commit/56505e340b3f84c2ab5b082de729c99d8b142061)

**Testing Guide**

Omitting --os-sku implicitly selects the Ubuntu container host, but explicitly setting it should still result in Ubuntu hosts being provisioned.
`az aks create -g resourcegroup -n clustername --os-sku Ubuntu`
`az aks nodepool add -g resourcegroup --cluster-name clustername -n newnodepool --os-sku Ubuntu`

Additionally I've verified that `--os-sku CBLMariner` is behaving as expected on a subscription with access to CBLMariner.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
